test/test_jobs/test_export_tasks.py

```diff
@@ -111,21 +110,21 @@ def test_export_tasks_csv_json(self, mail):
         attachment = message.attachments[0]
         assert attachment.filename == '{}_task_run_csv.zip'.format(filename)
 
-        fp = StringIO(message.attachments[0].data)
-        zfp = zipfile.ZipFile(fp, "r")
-        file_objects = zfp.infolist()
-        contents = [zfp.read(file_object) for file_object in file_objects]
-        expected_headers = [
-            'task_run__calibration', 'task_run__created',
-            'task_run__external_uid', 'task_run__finish_time',
-            'task_run__id', 'task_run__info', 'task_run__info__list',
-            'task_run__info__list__0__lastName', 'task_run__info__list__0__name',
-            'task_run__info__list__1__lastName', 'task_run__info__list__1__name',
-            'task_run__info__object', 'task_run__info__object__a',
-            'task_run__info__text', 'task_run__project_id', 'task_run__task__gold_answers',
-            'task_run__task_id', 'task_run__timeout', 'task_run__user_id',
-            'task_run__user_ip']
-        assert all(header in contents[0] for header in expected_headers)
+        # fp = StringIO(message.attachments[0].data)
```

Comment:
any particular reason to comment this test?

Reply:
I think it depends on the flatten feature which isn't ready yet. The whole test used to be disabled, but the part that I commented is the only part that would cause the test to fail. So I enabled the test and just commented that part out in order to increase test coverage.